pkg/controller/migmigration/migrate.go

```diff
@@ -90,11 +90,20 @@ func (r *ReconcileMigMigration) migrate(migration *migapi.MigMigration) (int, er
 		return 0, err
 	}
 
+	// TODO: SYNC WITH JEFF TO GET OPINION ON THIS HACK
+	// Setting this annotation to not create stage pods after copy restore has
+	// run to completion
+	if task.Phase == DeletingStagePods {
+		if migration.Annotations == nil {
+			migration.Annotations = make(map[string]string)
+		}
+		migration.Annotations["openshift.io/stage-completed"] = "true"
```

Comment:
This is the annotation currently used to track state for whether we've already created and deleted "stage pods" or "dummy pods". Will likely be rolled into a phase check soon.